### PR TITLE
Create function for update json file

### PR DESCRIPTION
**Pull request by dtrunov**
Create function for update json file

### CODE REVIEW

**Comment by dtrunov** [comment]
@VVeremjova please review and merge

File: user_modules/shared_testcases/commonFunctions.lua [hunk 1]
```diff
@@ -1179,5 +1180,52 @@ if file == nil then
...
+function commonFunctions:update_json_file(path_to_json, old_section, new_section)
+  local file = io.open(path_to_json, "r")
+  if file == nil then
+    print("File doesnt exist, path:"..path_to_json)
+    assert(false)
+  end
+  local json_data = file:read("*a")
+  file:close()
+  local data = json.decode(json_data)
+  local b, e = 0,0
```
Comment (VVeremjova) [inline user_modules/shared_testcases/commonFunctions.lua]: @dtrunov Please use more informative names or give additional comments

Comment (dtrunov) [inline user_modules/shared_testcases/commonFunctions.lua]: @VVeremjova Fixed it within commit 0443f5e

**Comment by dtrunov** [comment]
@VVeremjova Please review and merge

**Pull request merged**